test-asset-generator/transformers/transformer.js

```diff
@@ -44,13 +40,13 @@ export default class Transformer {
 
             switch (asset['@type']) {
                 case ASSET_TYPES.PERSONA:
-                    const personaTransformer = new PersonaTransformer(asset);
+                    const personaTransformer = new PersonaTransformer(asset, this.namespace);
                     transformedData = personaTransformer.transform();
                 case ASSET_TYPES.ORGANISATION:
```

Comment:
I know this isn't in your code, but how come there are no `break` statements in these switches? Does this work?

Reply:
Of course, you don't really need a break in a switch because the case will only ever match 1 value.